public/factory/todo.colors.js

```diff
@@ -0,0 +1,26 @@
+angular.module('todoApp')
```

Comment:
Why do we have fixed colors? Wouldn't it be better to use a color-picker to give freedom of color choosing?